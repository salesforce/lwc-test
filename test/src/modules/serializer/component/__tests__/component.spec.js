/*
 * Copyright (c) 2018, salesforce.com, inc.
 * All rights reserved.
 * SPDX-License-Identifier: MIT
 * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
 */
import { createElement } from 'lwc';
import Component from '../component';

it('serializes component with HTML', () => {
    const elm = createElement('serializer-component', { is: Component });
    document.body.appendChild(elm);

    expect(elm).toMatchSnapshot();
});
