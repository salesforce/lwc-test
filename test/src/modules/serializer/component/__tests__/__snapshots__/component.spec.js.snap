// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`serializes component with HTML 1`] = `
<serializer-component>
  #shadow-root(open)
    <h1>
      I am an LWC component
    </h1>
</serializer-component>
`;
